src/shim/server/shim.test.ts

```diff
@@ -134,102 +134,4 @@ test('hash class names', () => {
   ])
 })
 
-test('apply HTML parser options (comment: true)', () => {
```

Comment:
I think we should leave the test to ensure that comments are not removed. And maybe add one for the issue you had (the whitespace thing).

Reply:
Added back a html comment test in 962e44f.

I'm actually not sure how to get a minimal repo of the whitespace modifications issue I was seeing.

Reply:
I think we are good here.